src/main/resources/assets/minecolonies/gui/windowassigncitizen.xml

```diff
@@ -5,9 +5,9 @@
     <label size="100% 11" pos="0 50" textalign="TOP_MIDDLE" color="white" label="$(com.minecolonies.coremod.gui.assigning.description)"/>
     <list id="unassigned" size="70% 120" pos="60 70" align="TOP_MIDDLE">
         <view size="70% 40">
```

Comment:
this size_x 70% seems suspicious - the view should be 100% of the parent list, the list should have fixed size rather than 70% of window, also please remove the align and fix the pos accordingly

Reply:
*All* the aligns in this file?

Reply:
not "textalign"s, just "align"s - they are evil in fixed guis, you can skip the align part if you want to (I just hate them)

Reply:
I started removing most aligns when I touch certain UIs, the way it is currently handled I don't see a huge benefit to align.

Reply:
ye I'll remove it